src/main/java/org/hmhb/program/ProgramDao.java

```diff
@@ -0,0 +1,11 @@
+package org.hmhb.program;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface ProgramDao extends CrudRepository<Program, Long> {
+
+    List<Program> findByOrganizationId(Long organizationId);
```

Comment:
The magic here is that findByOrganizationId translates to "WHERE orgination_id = ?".

With spring-data, you get the basic crud operations (findAll, findOne, save, remove) for free, and anything else you put in your interface will be figured out at runtime and translated into the query.

So findByAbcAndXyz(int arg1, int arg2) would become WHERE <abc> = ? AND <xyz> = ?

With arg1 supplied to the first ?, and arg2 supplied to the second ?.

The <abc> and <xyz> would evaluate to whatever column is associated with the crud object's abc and xyz properties (the crud object being Program in our code, and it's organizationId being associated with organization_id in the DB).